### PR TITLE
Remove quizzes from config

### PR DESCRIPTION
**Pull request by LazarusAA**
This Pull Request closes issue #240. It includes the following changes:

- Removed quiz section from the config template
- Removed the quizzes fields from the config parser

This PR has been done as part of OnlyDust's ODHack 7.0

### CODE REVIEW

**Comment by LazarusAA** [comment]
Hi @Marchand-Nicolas

I was assigned this issue for ODHack 7.0. Please review and merge or request the required changes if required.

File: src/config.rs [hunk 1]
```diff
@@ -104,44 +104,6 @@ pub_struct!(Clone, Deserialize;  Discord {
...
-    {
-        let s = String::deserialize(deserializer)?;
-        match s.to_lowercase().as_str() {
-            "text_choice" => Ok(QuizQuestionType::TextChoice),
-            "image_choice" => Ok(QuizQuestionType::ImageChoice),
-            "ordering" => Ok(QuizQuestionType::Ordering),
-            _ => Err(serde::de::Error::custom("Unexpected type")),
-        }
-    }
-}
```
Comment (Marchand-Nicolas) [inline src/config.rs]: This should be kept

File: src/config.rs [hunk 2]
```diff
@@ -104,44 +104,6 @@ pub_struct!(Clone, Deserialize;  Discord {
...
-    correct_answers: Option<Vec<usize>>,
-    correct_order: Option<Vec<String>>,
-    image_for_layout: Option<String>,
-});
-
-pub_struct!(Clone, Deserialize,Debug; Quiz {
-    name: String,
-    desc: String,
-    questions: Vec<QuizQuestion>,
-});
```
Comment (Marchand-Nicolas) [inline src/config.rs]: This should also be kept

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]
Hello, thanks for your PR! This is almost perfect, but we want to keep the types (see bellow I indicated which parts of the code) because even if we don't read quizzes from the config anymore, we are now loading quizzes from the db and thus still use the types.

**Comment by LazarusAA** [comment]
Thanks for the feedback, it makes total sense. I just added the types back, please review and notify me if any changes are required.

**Review by Marchand-Nicolas — APPROVED** [review]
Perfect LGTM!

**Pull request merged**